test/testautomation_stdlib.c

```diff
@@ -4,6 +4,7 @@
 #include <SDL3/SDL.h>
 #include <SDL3/SDL_test.h>
 #include "testautomation_suites.h"
+#include "SDL_build_config.h"
```

Comment:
I would just drop these tests rather than conditionally test our implementation. We can just document the behavior as undefined, since it won't be consistent across platforms and C library implementations.

Reply:
OK, I removed them.